maestro/financial-connections/Testmode-PaymentIntent-TestInstitution-Networking.yaml

```diff
@@ -59,6 +59,11 @@ tags:
 - tapOn:
     id: "consent_cta"
 # LOGIN TO NETWORKING
+- assertVisible:
+    id: "existing_email-button"
+# Ensure the back button is not visible on the Link login bottom sheet
+- assertNotVisible:
+    id: "top-app-bar-back-button"
```

Comment:
Let’s also add the `assertVisible` for the same button in the OTP screen, so that we fully cover what was tested in the `ReturningUser.yaml` test that we’re getting rid of.

Reply:
https://github.com/stripe/stripe-android/pull/8050/commits/669baec00d987d1e3f44ff8e40358645023aeedf